From 0dff3ef8078285ab9723f521841df7349fa7b800 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Sun, 15 Jan 2017 16:56:55 +0200
Subject: [PATCH 1275/2241] fix: dts: mmc: a80x0: a70x0: remove unused
 parameter from SD/eMMC nodes

Parameter pad-type is relevant for Armada-3700, and should not be
used for Armada-7k/8k.

Change-Id: I7904c2bae183a69f91bdb951ad1e9f8a8d6dd6f6
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35590
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35588
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dtsi   | 1 -
 arch/arm64/boot/dts/marvell/armada-8040-db.dtsi   | 2 --
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 2 --
 3 files changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
index ffc8dc7..df1e774 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
@@ -112,7 +112,6 @@
 				bus-width = <4>;
 				no-1-8-v;
 				non-removable;
-				marvell,pad-type = "sd";
 			};
 			spi@700680 {
 				spi-flash@0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
index f439090..efd714f 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
@@ -119,8 +119,6 @@
 				bus-width = <4>;
 				no-1-8-v;
 				non-removable;
-				marvell,pad-type = "sd";
-				non-removable;
 			};
 		};
 
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 54b1a92..0d8fb56 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -72,7 +72,6 @@
 				no-1-8-v;
 				non-removable;
 				marvell,xenon-emmc;
-				marvell,pad-type = "fixed-1-8v";
 				status = "okay";
 			};
 			vccq_mmc_reg: regulator@0 {
@@ -103,7 +102,6 @@
 				bus-width = <4>;
 				no-1-8-v;
 				broken-cd;
-				marvell,pad-type = "sd";
 				pinctrl-names = "default";
 				pinctrl-0 = <&cpm_sdhci_pins>;
 				status = "okay";
-- 
2.7.4

